src/lib/SeamComponents.tsx

```diff
@@ -100,3 +108,13 @@ const createSeamContextValue = ({
 const defaultSeamContextValue = createDefaultSeamContextValue()
 
 export const seamContext = createContext<SeamContext>(defaultSeamContextValue)
+
+export function useSeamComponents() {
```

Comment:
If we revert the rename.
```suggestion
export function useSeamContext() {
```